src/animation/AnimationMixer.js

```diff
@@ -654,7 +654,7 @@ Object.assign( AnimationMixer.prototype, {
 
 		remove_empty_map: {
 
-			for ( var _ in bindingByName ) break remove_empty_map;
+			// for ( var _ in bindingByName ) break remove_empty_map;
```

Comment:
Ok i understand what's happen in this pretty piece of code, now !
I had never eared about javascript [labeled statement](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/label)...

So i will amend this commit as soon as possible !
And i make some bench on it versus this conditional code that do the same (notion):

```javascript
    if(Object.keys(obj).length === 0 && obj.constructor === Object) {
        // delete
    }
```

using labeled is 50% faster due to break on first iteration ! Really powerfull stuff !!!